mock/mock_test.go

```diff
@@ -131,20 +131,21 @@ func (i *TestExampleImplementation) TheExampleMethodFuncType(fn ExampleFuncType)
 
 // MockTestingT mocks a test struct
 type MockTestingT struct {
-	logfCount, errorfCount, failNowCount int
+	logfMessages, errorfMessages []string
+	failNowCount                 int
 }
 
 // Helper is like [testing.T.Helper] but does nothing.
 func (MockTestingT) Helper() {}
 
 const mockTestingTFailNowCalled = "FailNow was called"
 
-func (m *MockTestingT) Logf(string, ...interface{}) {
-	m.logfCount++
+func (m *MockTestingT) Logf(format string, a ...interface{}) {
+	m.logfMessages = append(m.logfMessages, fmt.Sprintf(format, a...))
```

Comment:
🎨 nitpicking here

They are no longer logfmessages if you used sprintf before adding string to them